actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -77,7 +95,7 @@ class ActionCable.Connection
       {identifier, message, type} = JSON.parse(event.data)
       switch type
         when message_types.welcome
-          @monitor.recordConnect()
+          break
```

Comment:
Why was this moved? It introduces a subtle behavior / timing change that doesn't seem relevant to your PR.